### DISCUSSION TITLE
iOS version from Podfile not taken into account

### DISCUSSION THREAD

**Discussion by TheCarpetMerchant** [codemagic-ci-cd/codemagic-docs#1576] [Q&A] [answered]
My Podfile has `platform :ios, '12.0'`, but I get this error : `Swift Compiler Error (Xcode): Compiling for iOS 11.0, but module 'workmanager' has a minimum deployment target of iOS 12.0: `

<details>
<summary>Full Podfile</summary>
# Uncomment the next line to define a global platform for your project
platform :ios, '12.0'

# CocoaPods analytics sends network stats synchronously affecting flutter build latency.
ENV['COCOAPODS_DISABLE_STATS'] = 'true'

project 'Runner', {
  'Debug' => :debug,
  'Profile' => :release,
  'Release' => :release,
}

def flutter_root
  generated_xcode_build_settings_path = File.expand_path(File.join('..', 'Flutter', 'Generated.xcconfig'), __FILE__)
  unless File.exist?(generated_xcode_build_settings_path)
    raise "#{generated_xcode_build_settings_path} must exist. If you're running pod install manually, make sure flutter pub get is executed first"
  end

  File.foreach(generated_xcode_build_settings_path) do |line|
    matches = line.match(/FLUTTER_ROOT\=(.*)/)
    return matches[1].strip if matches
  end
  raise "FLUTTER_ROOT not found in #{generated_xcode_build_settings_path}. Try deleting Generated.xcconfig, then run flutter pub get"
end

require File.expand_path(File.join('packages', 'flutter_tools', 'bin', 'podhelper'), flutter_root)

flutter_ios_podfile_setup

target 'Runner' do
  # Comment the next line if you don't want to use dynamic frameworks
  use_frameworks!
  use_modular_headers!

  flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))

  # Pods for Runner

end

post_install do |installer|
  installer.pods_project.targets.each do |target|
    flutter_additional_ios_build_settings(target)
  end
end
</details>

**Comment by icarusdust** [comment]
Hi @TheCarpetMerchant, please check your project targets' iOS Deployment Target in Xcode and confirm that they are all set to 12.0 or above.

**Reply by TheCarpetMerchant** [reply, marked as answer]
I'm on Windows, but you message made me think of looking for targets by runner profile. And indeed I found such a thing in the `project.pbxproj` file and it works. Thanks !
